### DISCUSSION TITLE
ANCR WG + 0PN - TopIC:  A Certification Program for Consent by Default

### DISCUSSION THREAD

**Discussion by smartopian** [KantaraInitiative/ancr-wg#20] [Ideas]
The GDPR and Conv 108+ - have been written from a consent by default perspective, and, if viewed this way, with common sense, digital privacy can be very easy-  with this in mind - as the point of Consent by Default, just a couple of work additions to Article 25, and the entire GDPR can work with a Consent by Default Certification program -  
- an 0PN - TransparencyLab - proposed - alternative program - which scales - as  an easy alternative approach with a consent (or control) by design and a data protection by default interpretation.  

- **Consent by Design and Data Protection by Default Certification**: for PII Controller DPO's Notice Credential,  which is used to implement a   consent notice record, receipt and token exchange service .   Providing security and evidence record information structure for digital identity based data governance and access infrastructure. 
 (All it would take is interpreting GDPR and Data Protection through the lens of a very slightly modified.

Article 25. 
Data protection by design and  by Default  
interpreted as **Consent** by design and **Data Protection** by Default

1. Taking into account the state of the art, the cost of implementation and the nature, scope, context and purposes of processing as well as the risks of varying likelihood and severity for rights and freedoms of natural persons posed by the processing, the controller shall, both at the time of the determination of the means for processing and at the time of the processing itself, implement appropriate technical and organisational measures, such as **(consent by design to govern the use of identifier technologies for)** pseudonymisation, which are designed to implement data-protection principles, such as data minimisation, in an effective manner and to integrate the necessary safeguards into the processing in order to meet the requirements of this Regulation and protect **(not replace, control access too, and restrict data, with non-standard consent)** the rights of data subjects**(, the personally identified individual)**.

2. The controller shall implement appropriate technical and organisational measures for ensuring that, by default, only personal data which are necessary for each specific purpose of the processing are processed. That obligation applies to the amount of personal data collected, the extent of their processing, the period of their storage and their accessibility. In particular, such measures shall ensure that by default personal data are not made accessible without the individual's intervention to an indefinite number of natural persons.

3. An approved certification mechanism pursuant to Article 42 may be used as an element to demonstrate compliance with the requirements set out in paragraphs 1 and 2 of this Article.

**Comment by IDmachines** [comment]
This is so very good. Thank you Mark.

 

Perhaps Consent by Default as intro to the Code of Conduct and Framework follow on to TPIs and reports.. 

 

 

From: Mark Lizar ***@***.***> 
Sent: Saturday, November 25, 2023 11:56 AM
To: KantaraInitiative/WG-ANCR ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [KantaraInitiative/WG-ANCR] ANCR WG + 0PN - TopIC: A Certification Program for Consent by Default (Discussion #20)

 

The GDPR and Conv 108+ - have been written from a consent by default perspective, and, if viewed this way, with common sense, digital privacy can be very easy- with this in mind - as the point of Consent by Default, just a couple of work additions to Article 25, and the entire GDPR works with a Consent by Default Certification program -

*	an 0PN - TransparencyLab - proposed - alternative program - which scales - as an easy alternative approach with a consent (or control) by design and a data protection by default interpretation.
*	Consent by Default Certification: for PII Controller DPO's Notice Credential, which is used to implement a consent notice record, receipt and token exchange service . Providing security and evidence record information structure for digital identity based data governance and access infrastructure.
(All it would take is interpreting GDPR and Data Protection through the lens of a very slightly modified.

Article 25.
Data protection by design and Consent by Default

1.	Taking into account the state of the art, the cost of implementation and the nature, scope, context and purposes of processing as well as the risks of varying likelihood and severity for rights and freedoms of natural persons posed by the processing, the controller shall, both at the time of the determination of the means for processing and at the time of the processing itself, implement appropriate technical and organisational measures, such as (consent by default governance framework with) pseudonymisation, which are designed to implement data-protection principles, such as data minimisation, in an effective manner and to integrate the necessary safeguards into the processing in order to meet the requirements of this Regulation and protect (not replace, control access too, and restrict) the rights of data subjects**(, the personally identified individual)**.
2.	The controller shall implement appropriate technical and organisational measures for ensuring that, by default, only personal data which are necessary for each specific purpose of the processing are processed. That obligation applies to the amount of personal data collected, the extent of their processing, the period of their storage and their accessibility. In particular, such measures shall ensure that by default personal data are not made accessible without the individual's intervention to an indefinite number of natural persons.
3.	An approved certification mechanism pursuant to Article 42 may be used as an element to demonstrate compliance with the requirements set out in paragraphs 1 and 2 of this Article.

**Comment by smartopian** [comment]
+1 - it needs some semantic testing - but I think with an edit, we could get all of this

The Roadmap for - Consent by Default Governance -

The  Digital Privacy Transparency Record Framework for PII Controllers is the over arching meme,.

 Consent  by Default Code of Conduct for Digital Privacy Expectations
1. If all parties do not have digital proof of  notice, consent is not valid
2. All privacy is concentric and requires a notice - even break the glass front door access, or break the glass back door,
- this means mass surveillance and mass secret surveillance by any party is illegal in the commons,

Logically, the Consent by Default ANCR Roadmap (could be)

1. The PII Controller Notice Identity & Credential
2. The Digital Privacy Transparency Scheme
3. The Consent by Default Code of Conduct & Record Framework
4. AuthC Exchange Protocol for the * Consent (Notice) Receipt V2: The Consent Token
5. Digital Commons Interop - Starting with NIST

Interop - Alternative (to surveillance capitalism) profiles, accept digital consent tokens instead
Digital Consent by Default Alternative -
Start by Registering  as PII Controller

Shall  we call a vote in Dec:  for
1. New Charter and roadmap,
2,.  consensus protocol for ANCR WG
3,  Leadership for 2024 -




On 26 Nov 2023, at 08:57, IDmachines ***@***.***> wrote:



This is so very good. Thank you Mark.



Perhaps Consent by Default as intro to the Code of Conduct and Framework follow on to TPIs and reports..





From: Mark Lizar ***@***.***>
Sent: Saturday, November 25, 2023 11:56 AM
To: KantaraInitiative/WG-ANCR ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [KantaraInitiative/WG-ANCR] ANCR WG + 0PN - TopIC: A Certification Program for Consent by Default (Discussion #20)



The GDPR and Conv 108+ - have been written from a consent by default perspective, and, if viewed this way, with common sense, digital privacy can be very easy- with this in mind - as the point of Consent by Default, just a couple of work additions to Article 25, and the entire GDPR works with a Consent by Default Certification program -

* an 0PN - TransparencyLab - proposed - alternative program - which scales - as an easy alternative approach with a consent (or control) by design and a data protection by default interpretation.
* Consent by Default Certification: for PII Controller DPO's Notice Credential, which is used to implement a consent notice record, receipt and token exchange service . Providing security and evidence record information structure for digital identity based data governance and access infrastructure.
(All it would take is interpreting GDPR and Data Protection through the lens of a very slightly modified.

Article 25.
Data protection by design and Consent by Default

1. Taking into account the state of the art, the cost of implementation and the nature, scope, context and purposes of processing as well as the risks of varying likelihood and severity for rights and freedoms of natural persons posed by the processing, the controller shall, both at the time of the determination of the means for processing and at the time of the processing itself, implement appropriate technical and organisational measures, such as (consent by default governance framework with) pseudonymisation, which are designed to implement data-protection principles, such as data minimisation, in an effective manner and to integrate the necessary safeguards into the processing in order to meet the requirements of this Regulation and protect (not replace, control access too, and restrict) the rights of data subjects**(, the personally identified individual)**.
2. The controller shall implement appropriate technical and organisational measures for ensuring that, by default, only personal data which are necessary for each specific purpose of the processing are processed. That obligation applies to the amount of personal data collected, the extent of their processing, the period of their storage and their accessibility. In particular, such measures shall ensure that by default personal data are not made accessible without the individual's intervention to an indefinite number of natural persons.
3. An approved certification mechanism pursuant to Article 42 may be used as an element to demonstrate compliance with the requirements set out in paragraphs 1 and 2 of this Article.
